loop/README.md

```diff
@@ -0,0 +1,50 @@
+# Tests for Loop
+
+##General Configuration
+
+It's highly recommended to use [Virtualenv](https://virtualenv.pypa.io/en/latest/)
+in order to have an isolated environment.
+
+From /loop
+
+1. `virtualenv venv`
+2. `source venv/bin/activate`
+3. `pip install -r requirements.txt`
+
+Verify that the `manifest.ini` file contains expected values since the tests
+rely on using it for assertions.
+
+## Configuration Check Tests
+
+These tests are to be run whenever a deployment of loop happens. They access
```

Comment:
atm these can't be run in automation, so you may want to specify that these must be run manually